app/javascript/react/src/containers/IndexPage.js

```diff
@@ -23,30 +27,61 @@ class IndexPage extends Component{
       })
       .then(response => response.json())
       .then(body => {
-        this.setState({celestials_array: body})
+        this.setState({celestialsArray: body})
       })
       .catch(error => console.error(`Error in fetch: ${error.message}`));
+  }
 
+  handleChangeDisplay(celestial_type){
+    this.setState({selected_type: celestial_type})
   }
 
   render() {
+    console.log(this.state.selected_type)
+    let celestials = this.state.celestialsArray.map(celestial =>{
+      if(this.state.selected_type == ''){
+        return(
+          <CelestialTile
+            key = {celestial.id}
+            id = {celestial.id}
+            name = {celestial.name}
+            type = {celestial.celestial_type}
+            photo = {celestial.photo}
+          />
+        )
+      } else if (this.state.selected_type == celestial.celestial_type){
```

Comment:
This conditional is currently not doing anything. In both cases, you are rendering the same component with the same props. I would clean this up if you are not using it, or move it to another feature branch if you are planning to use this in the future.